packages/app/src/app/overmind/namespaces/editor/actions.ts

```diff
@@ -220,6 +220,13 @@ export const sandboxChanged: AsyncAction<{ id: string }> = withLoadApp<{
 
         return aggr;
       }, {});
+
+      if (effects.router.getCommentId()) {
+        actions.workspace.setWorkspaceItem({ item: 'comments' });
+        actions.comments.selectComment({
+          commentId: effects.router.getCommentId(),
+        });
```

Comment:
```suggestion
        actions.comments.selectComment({
          commentId: effects.router.getCommentId(),
          bounds: {
            left: effects.browser.getWidth() / 2,
            top: 0,
            right: effects.browser.getWidth() / 3,
            bottom: 0
          }
        });
```

By passing in the trigger bounds here we are more explicit about when this scenario actually occurs, which is when we grab a comment id from the url 😄